app/views/provisioned.html

```diff
@@ -0,0 +1,156 @@
+                      <!--<div class="list-pf-details">
+                        <div ng-if="!row.expanded">
+                          <div class="hidden-xs hidden-sm">
+                            <span ng-if="!row.bindings.length
+                                        && row.isBindable
+                                        && ({resource: 'bindings', group: 'servicecatalog.k8s.io'} | canI : 'create')">
+                              <a href="" ng-click="row.showOverlayPanel('bindService', {target: row.apiObject})">Create Binding</a>
+                            </span>
+                            <span ng-if="row.bindings.length" class="component-label">Bindings</span>
+                            <p ng-if="firstBinding = row.bindings[0]" class="bindings">
+                              <span ng-if="application = row.state.applicationsByBinding[firstBinding.metadata.name][0]">
+                                {{application.metadata.name}}
+                              </span>
+                              <span ng-if="!application">
+                                {{firstBinding.spec.secretName}}
+                              </span>
+                              <span ng-if="row.bindings.length > 1">
+                                and
+                                <a
+                                  ng-if="row.bindings.length > 1"
+                                  ng-click="row.toggleExpand($event, true)">
+                                  {{row.bindings.length -1}} other<span ng-if="row.bindings.length > 2">s</span></a>
+                              </span>
+                            </p>
+                          </div>
+                        </div>-->
+
+                      <!--{{overview.bindingsByInstanceRef[service.metadata.name]}}-->
+                      <!--<p ng-if="firstBinding = overview.bindingsByInstanceRef[service.metadata.name][0]" class="bindings">
+                        <span ng-if="application = row.state.applicationsByBinding[firstBinding.metadata.name][0]">
+                          {{application.metadata.name}}
+                        </span>
+                                  <span ng-if="!application">
+                          {{firstBinding.spec.secretName}}
+                        </span>
+                                  <span ng-if="row.bindings.length > 1">
+                          and
+                          <a
+                            ng-if="row.bindings.length > 1"
+                            ng-click="row.toggleExpand($event, true)">
+                            {{row.bindings.length -1}} other<span ng-if="row.bindings.length > 2">s</span></a>
+                        </span>
+                      </p>-->
+                    </td>
+                    <td data-title="Status">
+                      <div row class="status">
+                        <status-icon status="serviceInstance | deploymentStatus" disable-animation></status-icon>
```

Comment:
`deploymentStatus` is not correct here. You'll need to look at the overview service instance row to see how we determine status.